src/parse.rs

```diff
@@ -1482,18 +1486,24 @@ impl InlineStack {
                 min(mod3_lower, self.lower_bounds[1])
             }
         } else {
-            0
+            self.lower_bounds[5]
```

Comment:
Might be a good idea to define constants for these magic numbers, now that there are a few of them.

Reply:
:+1: done!